.github/workflows/rust.yml

```
@@ -0,0 +1,58 @@
+name: rust
+
+on:
+  pull_request: {}
+  push:
+    branches: develop
+
+env:
+  CARGO_INCREMENTAL: 0
+  RUSTFLAGS: "-Dwarnings"
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rust:
+          - 1.41.0 # MSRV
```

Can drop this down to 1.36.0 for now

```suggestion
          - 1.36.0 # MSRV
```